package day0202;

public class UseRuntimeException {

	public static void main(String[] args) {

		
		
		
		
		
		
		//main method의 arguments를 두개 입력받아 나눈 연산을 수행한 후
		//결과를 보여주는 일.
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
	}//main

}//class
